helpick-service/src/main/java/com/infoshare/database/FileDb.java

```diff
@@ -77,6 +78,42 @@ public void saveVolunteer(Volunteer volunteer) {
 
     }
 
+    @Override
+    public void saveVolunteerWithUuid(Volunteer volunteer){
+        List<Volunteer> allVolunteers = getVolunteers();
```

Comment:
Może warto rozważyć trzymanie listy wolontariuszy w pamięci? Bo w tej chwili cały czas wczytujemy ich z pliku i przez to jest sporo kłopotów, np. z edycją.
Jeśli lista trzymana byłaby w pamięci, to edytujemy obiekt w pamięci, a operacja `save` robiłaby tylko `zrzut` całej listy do pliku, żeby mieć również w pliku aktualne dane.

Reply:
ten pomysł też podam jako temat do rozmowy na projekcie :)